.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ env:
 
 jobs:
   test-report:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Latest Ubuntu LTS

Reply:
I've reverted this change, since Ubuntu 22.04 is still not available for GitHub actions.